### DISCUSSION TITLE
Digits not normalized?

### DISCUSSION THREAD

**Discussion by nestyme** [openai/whisper#955] [Q&A]
Hello! Is it possible to get digits/abbreviations written by letters?
I understand that it can be done with logits manually but maybe there is easier way. I do know about denormalisation libraries e.g. num2words or NeMo num2word tool, but it doesn't use any sound info which can result in mistakes.
Thank you!
